releasing/VERSIONS

```diff
@@ -6,7 +6,7 @@
 # kyaml version
 export kyaml_major=0
 export kyaml_minor=0
-export kyaml_patch=1
+export kyaml_patch=2
```

Comment:
Are the version in this file for the next release versions? Should it be reviewed first before cutting the release?

Reply:
the tooling respects either.  up to us what we want our process to be.